deploy/kubernetes/base/controller-serviceaccount.yaml

```diff
@@ -47,7 +47,7 @@ metadata:
 subjects:
   - kind: ServiceAccount
     name: efs-csi-controller-sa
-    namespace: default
+    namespace: sample
```

Comment:
this change seems unintentional, i think it will break the kustomize install command we put in the README

Reply:
I will fix it in a future pr, might be that make generate-kustomize has an issue